### PR TITLE
Set up express server and seeded db

### PR DESCRIPTION
**Pull request by rlnolan15**

### CODE REVIEW

**Comment by rlnolan15** [comment]
@jonyang221 @jshamoo @Evillivekai @j-gens Hey could someone checkout my express server and seeding method? Thx

File: package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    "nodemon": "nodemon server/index.js"
+  },
+  "dependencies": {
+    "axios": "^0.19.0",
+    "body-parser": "^1.19.0",
+    "express": "^4.17.1",
+    "mongoose": "^5.7.4",
+    "mongoose-auto-increment": "^5.0.1",
+    "mongoose-data-seed": "^2.1.4",
+    "mongoose-seed": "^0.6.0",
```
Comment (j-gens) [inline package.json]: you may want to clean these out of your dependencies since you're not using them :)

File: package.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  "description": "",
+  "author": "",
+  "main": "index.js",
+  "license": "",
+  "engines": {
+    "node": ">=6.13.0"
+  },
+  "scripts": {
+    "seed": "node server/seeder.js",
+    "build": "webpack --config webpack.config.js",
```
Comment (jshamoo) [inline package.json]: If you have a webpack.config.js, webpack will run that config file automatically, so you don't need to specify in the script. Not a big deal. Just FYI :)

File: server/index.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+app.get("/", (req, res) => {
+  res.send("Heyyyyy");
+});
+
+app.listen(port, () => {
+  console.log(`App listening on port ${port}`);
+});
+
+module.exports.app = app;
```
Comment (jshamoo) [inline server/index.js]: I am curious why you are exporting your app?

File: package.json (hunk 1 above)
Comment (rlnolan15) [reply package.json]: Good looks! Thanks.

File: package.json (hunk 2 above)
Comment (rlnolan15) [reply package.json]: Oh really? Sweeeeet!

File: server/index.js (hunk 3 above)
Comment (rlnolan15) [reply server/index.js]: Oh, I can't remember. I must've changed my plans. Thanks! I don't need to export it.

File: server/npm-debug.log [hunk 4]
```diff
@@ -0,0 +1,48 @@
+0 info it worked if it ends with ok
```
Comment (therobinkim) [inline server/npm-debug.log]: .log files should not be committed

**Pull request merged**